source/developers-guide/rest-api/api-resource-orders/index.md

```diff
@@ -86,6 +86,54 @@ This API call requires one of the following parameters to be defined:
 | paymentStatusId       | integer (foreign key) | **[Status](../models/#payment-status)**                                       |
 | orderStatusId         | integer (foreign key) | **[OrderStatus](../models/#order-status)**                                    |
 
+## GET (List)
+
+#### Required Parameters
+
+For this operation, no parameters are required. To get a list of all orders, simply query:
+* **http://my-shop-url/api/orders**
+
+### Return Value
+
+| Model                              | Table                 |
+|------------------------------------|-----------------------|
+| Customer  | s_user                |
```

Comment:
The Order-endpoint returns "Order" objects coming from the s_order-table. And yes, the documentation for retrieving a single order is wrong as well. So if you want to extend your PR to include a fix for that error, it would be very much appreciated! 😄